src/mumble/LookConfig.ui

```diff
@@ -447,17 +447,36 @@
         </property>
        </widget>
       </item>
-      <item row="2" column="0" colspan="2">
-       <widget class="QCheckBox" name="qcbAskOnQuit">
+      <item row="2" column="0">
+       <widget class="QLabel" name="qliConnectedQuitBehaviour">
+        <property name="text">
+         <string>Quit Behaviour</string>
+        </property>
+       </widget>
+      </item>
+      <item row="2" column="1">
+       <widget class="MUComboBox" name="qcbConnectedQuitBehaviour">
         <property name="toolTip">
-         <string>Ask whether to close or minimize when quitting Mumble.</string>
+         <string>This setting controls the behaviour of clicking on the X in the top right corner when connected.</string>
         </property>
         <property name="whatsThis">
-         <string>&lt;b&gt;If set, will verify you want to quit if connected.&lt;/b&gt;</string>
-        </property>
-        <property name="text">
-         <string>Ask on quit while connected</string>
+         <string>This setting controls the behaviour of clicking on the X in the top right corner when connected to a server. If you select &lt;i&gt;Ask&lt;/i&gt; a dialog will ask you what the application should do. &lt;i&gt;Minimize&lt;/i&gt; will only minimize the application. &lt;i&gt;Close&lt;/i&gt; will close the application.</string>
```

Comment:
```suggestion
         <string>This setting controls the behavior of clicking on the X in the top right corner when connected to a server. If you select &lt;i&gt;Ask&lt;/i&gt; a dialog will ask you what the application should do. &lt;i&gt;Minimize&lt;/i&gt; will minimize the application instead of closing it and &lt;i&gt;Close&lt;/i&gt; will close the application without asking for confirmation.</string>
```